docs/development.md

```diff
@@ -39,23 +39,31 @@ In addition, the command will also *minify* the JavaScript files to reduce the s
 
 #### Starting the dev server
 
-Run the following command:
+To start the server in the background, run the following command 
+and wait until the task exits with a `BUILD SUCCESSFUL`:
 
 ```sh
 ./gradlew appengineRun
 ```
+To start the server in the foreground (e.g. if you want the console output to be visible),
+run the following command instead: 
+```sh
+./gradlew appengineRun -Pdisable_daemon
+```
 
-Wait until the task exits with a `BUILD SUCCESSFUL`.
 The dev server URL will be `http://localhost:8888` as specified in `build.gradle`.
 
 #### Stopping the dev server
 
-Run the following command:
+If you started the server in the background, run the following command to stop it: 
 
 ```sh
 ./gradlew appengineStop
 ```
 
+If the server is running in the foreground, press `Ctrl + C` to stop it.
+
+
```

Comment:
Delete this blank line (throughout the rest of the document, only a single blank line separates sections).